vllm/entrypoints/llm.py

```diff
@@ -806,24 +806,26 @@ def encode(
 
     def score(
         self,
-        query: SingletonPrompt,
         texts: Union[SingletonPrompt, Sequence[SingletonPrompt]],
+        text_pairs: Union[SingletonPrompt, Sequence[SingletonPrompt]],
```

Comment:
Maybe `text_1` and `text_2`? `text_pairs` is a bit confusing to me as it suggests that I should pass in a list of tuples.